cvat/apps/documentation/user_guide.md

```diff
@@ -373,31 +468,96 @@ It is possible to handle many objects on the same frame in the mode.
 
 ![](static/documentation/images/image058.jpg)
 
-It is more convenient to annotate objects of the same type. For the purpose it is possible to specify a corresponding filter. For example, the following filter will hide all objects except pedestrians: ``//pedestrian``.
+It is more convenient to annotate objects of the same type. For the purpose it is possible to specify a corresponding filter. For example, the following filter will hide all objects except pedestrians: ``pedestrian``.
 
 To navigate between objects (pedestrians in the case) use the following shortcuts:
-- ``Tab`` - go to the next object
-- ``Shift+Tab`` - go to the previous object.
+- ``Tab`` — go to the next object
+- ``Shift+Tab`` — go to the previous object.
 
 By default in the mode objects are zoomed. To disable the functionality uncheck the corresponding setting: ``Open Menu`` —> ``Settings`` —> ``Zoom boxes in Attribute Annotation Mode``.
 
-By default other objects are hidden. To change the behaviour uncheck the corresponding setting: ``Open Menu`` —> ``Setting`` —> ``Hide Other in Attribute Annotation Mode``.
+By default other objects are hidden. To change the behavior uncheck the corresponding setting: ``Open Menu`` —> ``Setting`` —> ``Hide Other in Attribute Annotation Mode``.
+
+## Annotation with polygons
+
+It is used for semantic / instance segmentation.
+
+Be sure ``Z-Order`` flag in ``Create task`` dialog is enabled if you want annotate polygons.
+
```

Comment:
We need add:
Z-Order flag defines order of drawing. It need in order to get right annotation mask without extra work (additional drawing of borders).

Reply:
And we can switch it by '+' and '-' shortkeys.

Reply:
Which set maximum and minimum z-order respectively.